fv3config/cli.py

```diff
@@ -17,10 +17,63 @@ def _parse_args():
     return parser.parse_args()
 
 
+def _parse_enable_restart_args():
+    parser = argparse.ArgumentParser("enable_restart")
+    parser.add_argument(
+        "config",
+        help="URI to fv3config yaml file. Supports any path used by fsspec. "
+        "File will be modified in place.",
+    )
+    parser.add_argument(
+        "--initial-conditions",
+        help="Path to restart initial conditions.",
+        default=None,
```

Comment:
By default this routine is going to write a config.yaml which says initial_conditions is None. Please either modify the code so it doesn't modify the initial conditions by default, or make this a required argument.

Reply:
I actually don't think this was true, but regardless, initial conditions is now a required argument of the `enable_restart` function and a required positional argument of the CLI.

Reply:
OK, sorry for the misread and thanks for the update.